src/main/java/net/mcreator/element/converter/ConverterRegistry.java

```diff
@@ -70,8 +70,16 @@
 		put(ModElementType.POTION, Collections.singletonList(new PotionToEffectConverter()));
 	}};
 
+	// Converters that convert older mod element type to a newer one
+	private static final Map<String, IConverter> converters_legacy = new HashMap<>() {{
+	}};

```

Comment:
What is the purpose of subclassing HashMap here with {{}}?

Reply:
It is placeholder for converters, eg. for food me Goldorion will prepare.